packages/millicast-viewer-demo/src/viewer.js

```diff
@@ -120,7 +122,19 @@ const addStream = (stream) => {
       video.removeAttribute("autoplay");
     }
 
-    video.srcObject = stream;
+    //If we already had a a stream
+    if (video.srcObject) {
+       //Create temporal video element and switch streams when we have valid data
+       const tmp = document.createElement("video")
+       tmp.srcObject = stream;
+       tmp.addEventListener('loadedmetadata', (event) => {
+         Logger.log("loadedmetadata tmp",event);
+         video.srcObject = stream;
```

Comment:
this allows to change the stream only when we have playable video on the stream, could be interesting to be able to notify the sdk that it can stop the previous signaling now without having to wait for the timeout